staking/app/StakeConnection.ts

```diff
@@ -56,6 +50,12 @@ import {
 import assert from "assert";
 import { PositionAccountJs } from "./PositionAccountJs";
 import * as crypto from "crypto";
+import {
+  PriorityFeeConfig,
+  TransactionBuilder,
+  sendTransactions,
+} from "@pythnetwork/solana-utils";
+import NodeWallet from "@coral-xyz/anchor/dist/cjs/nodewallet";
```

Comment:
is this import correct? usually it's wrong to import from `dist` directory.

Reply:
This is a hack to work between two versions of `@coral-xyz/anchor`